kubetest/anywhere.go

```diff
@@ -107,6 +113,15 @@ func newKubernetesAnywhere(project, zone string) (deployer, error) {
 		zone = "us-central1-c"
 	}
 
+	kubeletVersion := *kubernetesAnywhereKubeletVersion
```

Comment:
Instead of having one version flag override the other, should we just make them mutually-exclusive and print an error if both are specified? Do we have a usecase where we actually want to specify both and have one be ignored?